web/components/contract/contracts-list.tsx

```diff
@@ -64,9 +65,9 @@ function DesktopPopover(props: { contract: Contract }) {
         />
       }
     >
-      <span tabIndex={0}>
```

Comment:
@akrolsmir I am not sure how this `tabIndex={0}` got here, but it was breaking "tab through the search results".